src/module.c

```diff
@@ -11763,6 +11786,20 @@ static uint64_t moduleEventVersions[] = {
  *
  *         ValkeyModuleKey *key;    // Key name
  *
+ * * ValkeyModule_Authentication_Attempt
```

Comment:
This seems wrong but I'm not sure which struct or value is referenced here. (Maybe other event names in this comment are wrong too?)

```suggestion
 * * ValkeyModuleEvent_AuthenticationAttempt
```

Reply:
I think it's supposed to be the define enum, `VALKEYMODULE_EVENT_AUTHENTICATION_ATTEMPT`. I'm not sure why these aren't in all CAPs.

Reply:
We should fix it...

Reply:
@zuiderkwast should I fix the whole comment, including the above event names?

Reply:
I fixes the function comment. The names there match the list of `ValkeyModuleEvent` variables declared in valkeymodule.h